scripts/deploy-app.sh

```diff
@@ -27,37 +27,43 @@ else
   VERSION=${GIT_SHA:0:7}
 fi
 
+source scripts/image_vars.src
+generateImageArgs(){
+  local _IMAGE_TAG=$2
+
+  case "$1" in
+    dev|qa) IMG_NAME_PREFIX="${1}_";;
+    sandbox|prod|prod-test) IMG_NAME_PREFIX="";;
+    *) { echo "Unknown environment: $1"; exit 20; }
+  esac
+
+  for PREFIX in ${VAR_PREFIXES_ARR[@]}; do
+    local HELM_KEY=`getVarValue ${PREFIX} _HELM_KEY`
+    local IMAGE_NAME=${IMG_NAME_PREFIX}`getVarValue ${PREFIX} _IMG`
+    echo "--set-string images.$HELM_KEY.tag=${_IMAGE_TAG} "
+    echo "--set-string images.$HELM_KEY.imageName=${IMAGE_NAME} "
```

Comment:
This addresses the "unnecessary quotes" problem that resulted in code be reverted in #599.